.ci/static.sh

```diff
@@ -12,9 +15,24 @@ function usage {
 }
 
 if [[ "$COMMAND" == "install" ]]; then
+    conda install jupyter
     pip install codespell flake8 pylint
 elif [[ "$COMMAND" == "run" ]]; then
-    flake8 -v nengo && codespell -q 3 && pylint nengo
+    # Convert notebooks to Python scripts
+    jupyter-nbconvert \
+        --log-level WARN \
+        --to python \
+        --TemplateExporter.exclude_input_prompt=True \
+        -- **/*.ipynb
+    # Remove style issues introduced in the conversion:
+    #   s/# $/#/g replaces lines with just '# ' with '#'
+    #   /get_ipython()/d removes lines containing 'get_ipython()'
+    sed -i -e 's/# $/#/g' -e '/get_ipython()/d' -- docs/**/*.py
+    flake8 nengo || STATUS=1
+    flake8 --ignore=E703,W291,W391 docs || STATUS=1
```

Comment:
I've made a commit that makes all the notebooks compliant. I've also updated `devscripts/clearoutputs.py` to remove trailing whitespace, so it should be relatively easy for developers to clean new notebooks. I haven't actually enabled the check yet, though, since I want to check with @tbekolay that there's not another reason not to do this.

Reply:
Yeah the only reason to keep it was because of the comment above, that lines with `# ` instead of `#` are inserted, but I fixed that in the `sed` statement, so I'll reenable this check.